### ISSUE TITLE
Match Close/Minimize buttons for older versions of Windows

### ISSUE THREAD

**Issue by phillip-haydon** [fluentCODE/ScrawlIssues#151]
This is a subjective opinion issue I thought I would raise. 

Feel free to close if you disagree :)

![image](https://cloud.githubusercontent.com/assets/895629/7562055/c186a8aa-f803-11e4-9178-d82112c4e745.png)

Top is spotify, bottom is scrawl. 

However, hovers are much nicer in Scrawl. 

![scrawl-vs-spotify](https://cloud.githubusercontent.com/assets/895629/7562082/00d14c04-f804-11e4-878d-bcd1afe73557.gif)

I think the icons look nicer in Spotify, but the hovers look better in scrawl. 

IMO.

**Comment by raymens** [comment]
I agree, but for me it would me more important to align the icons, menu items and the update text better

**Comment by stirno** [comment]
Our icons are intended to be the same as Win10.. which changes a bit every now and then but we're still very close.
![screen shot 2015-05-11 at 12 20 16 pm](https://cloud.githubusercontent.com/assets/666175/7570746/1003717e-f7d8-11e4-8ef4-63586abb9095.png)

**Comment by pauljz** [comment]
Early on we'd talked a bit about customizing this depending on what Windows version you're on, but it's enough effort that I'm not sure we wanna do that yet.

Windows unfortunately doesn't have an API for "give me the damn buttons that you use everywhere" when you've otherwise customized the window chrome.
